.gitignore

```diff
@@ -377,4 +377,5 @@ FodyWeavers.xsd
 # Additional files built by Visual Studio
 *.tlog
 
-# End of https://www.toptal.com/developers/gitignore/api/visualstudio
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/visualstudio
+docs/reference/tc_5-31_-_vc_boobytraps_text.pdf
```

Comment:
What's this?

Reply:
A period US army manual on VC trap tactics I found, planning to use it as a reference for future iterations of the trap script

Reply:
That's really cool.